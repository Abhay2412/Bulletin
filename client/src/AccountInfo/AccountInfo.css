.AccountInfo {
  text-align: center;
}

.Account-background {
  background-color: white;
  min-height: 100vh;
  display: flex;
  flex-direction: row;
  align-items: center;
  justify-content: left;
}

.account {
  position: absolute;
  width: 154px;
  height: 36px;
  left: 527px;
  left: calc(100vw / 2);
  top: 130px;

  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
    Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
  font-style: normal;
  font-weight: 700;
  font-size: 30px;
  line-height: 35px;

  color: #68390d;
}

.password {
  position: absolute;
  width: 170px;
  height: 21px;
  left: calc((100vw / 2));
  top: 230px;

  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
    Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
  font-style: normal;
  font-weight: 400;
  font-size: 14px;
  line-height: 16px;

  color: #68390d;
}

.new-password {
  position: absolute;
  width: 170px;
  height: 21px;
  left: calc(100vw / 2);
  top: 330px;

  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
    Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
  font-style: normal;
  font-weight: 400;
  font-size: 14px;
  line-height: 16px;

  color: #68390d;
}

.phone {
  position: absolute;
  width: 170px;
  height: 18px;
  left: calc((100vw / 2));
  top: 430px;

  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
    Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
  font-style: normal;
  font-weight: 400;
  font-size: 14px;
  line-height: 16px;

  color: #68390d;
}

.email {
  position: absolute;
  width: 170px;
  height: 15px;
  left: calc((100vw / 2));
  top: 530px;

  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
    Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
  font-style: normal;
  font-weight: 400;
  font-size: 14px;
  line-height: 16px;

  color: #68390d;
}

.alt-email {
  position: absolute;
  width: 170px;
  height: 15px;
  left: calc((100vw / 2));
  top: 630px;

  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
    Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
  font-style: normal;
  font-weight: 400;
  font-size: 14px;
  line-height: 16px;

  color: #68390d;
}

.side-panel {
  position: absolute;
  text-align: center;
  height: 100vh;
  width: 20vw;
  background: #cbdae1;
}

.logo {
  position: relative;
  width: 100px;
  height: 100px;
  top: 130px;
}

.nav {
  position: relative;
  top: 160px;

  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
    Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
  font-style: normal;
  font-weight: 400;
  font-size: 14px;
  line-height: 16px;

  color: #68390d;
}

ul {
  margin: 1;
  padding: 0;
}

li a {
  display: block;
  color: #68390d;
  padding: 8px 16px;
  text-decoration: none;
}

li a.active {
  background-color: #ecd1c2;
  color: #68390d;
}

li a:hover {
  background-color: #ecd1c2;
  color: #68390d;
}

.profile-pic {
  position: relative;
  width: 135px;
  height: 135px;
  top: 133px;
  left: calc(20vw / 2);
  border-radius: 75px;
}

.account-panel {
  position: absolute;
  text-align: center;
  /* justify-content: center;
  align-items: center; */
  height: 100vh;
  width: 45vw;
  background: #62808e;
}

.name {
  position: absolute;
  width: 121px;
  height: 18px;
  top: 275px;
  left: calc((20vw / 2) + 130px);

  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
    Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
  font-style: normal;
  font-weight: 700;
  font-size: 24px;
  line-height: 28px;

  color: #e3d4c6;
}

.last-password {
  position: absolute;
  width: 230px;
  height: 21px;
  top: 325px;
  left: calc((40vw / 2) + 80px);

  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
    Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
  font-style: normal;
  font-weight: 400;
  font-size: 14px;
  line-height: 16px;

  color: #e3d4c6;
}
