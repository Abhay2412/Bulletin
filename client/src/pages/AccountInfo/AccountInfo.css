.AccountInfo {
  text-align: center;
}

.Account-background {
  background-color: white;
  min-height: 100vh;
  display: flex;
  flex-direction: row;
  align-items: center;
  justify-content: left;
}

.account {
  position: absolute;
  width: 154px;
  height: 36px;
  left: 527px;
  left: calc((100vw / 1.5) - 10px);
  top: 100px;

  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
    Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
  font-style: normal;
  font-weight: 700;
  font-size: 30px;
  line-height: 35px;

  color: #68390d;
}

.password {
  position: absolute;
  width: 170px;
  height: 21px;
  left: calc((100vw / 1.5) - 70px);
  top: 200px;

  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
    Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
  font-style: normal;
  font-weight: 400;
  font-size: 14px;
  line-height: 16px;

  color: #68390d;
}

.new-password {
  position: absolute;
  width: 170px;
  height: 21px;
  left: calc((100vw / 1.5) - 70px);
  top: 300px;

  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
    Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
  font-style: normal;
  font-weight: 400;
  font-size: 14px;
  line-height: 16px;

  color: #68390d;
}

.phone {
  position: absolute;
  width: 170px;
  height: 18px;
  left: calc((100vw / 1.5) - 77px);
  top: 400px;

  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
    Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
  font-style: normal;
  font-weight: 400;
  font-size: 14px;
  line-height: 16px;

  color: #68390d;
}

.email {
  position: absolute;
  width: 170px;
  height: 15px;
  left: calc((100vw / 1.5) - 71px);
  top: 500px;

  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
    Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
  font-style: normal;
  font-weight: 400;
  font-size: 14px;
  line-height: 16px;

  color: #68390d;
}

.alt-email {
  position: absolute;
  width: 170px;
  height: 15px;
  left: calc((100vw / 1.5) - 60px);
  top: 600px;

  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
    Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
  font-style: normal;
  font-weight: 400;
  font-size: 14px;
  line-height: 16px;

  color: #68390d;
}

.input[type="text"] {
  position: absolute;
  width: 245px;
  height: 51px;
  left: calc((100vw / 1.5) - 55px);
  top: 235px;

  box-sizing: border-box;
  border: 0px;
  background: #cbdae1;
  outline: none;

  color: black;
  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
    Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
  font-style: normal;
  font-weight: 500;
  font-size: 14px;
  line-height: 16px;
}

.input[type="text"]:focus {
  border: 2px solid #ecd1c2;
}

.input1[type="text"] {
  position: absolute;
  width: 245px;
  height: 51px;
  left: calc((100vw / 1.5) - 55px);
  top: 335px;

  box-sizing: border-box;
  border: 0px;
  background: #cbdae1;
  outline: none;

  color: black;
  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
    Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
  font-style: normal;
  font-weight: 500;
  font-size: 14px;
  line-height: 16px;
}

.input1[type="text"]:focus {
  border: 2px solid #ecd1c2;
}

.input-phone[type="text"] {
  position: absolute;
  width: 245px;
  height: 51px;
  left: calc((100vw / 1.5) - 55px);
  top: 435px;

  box-sizing: border-box;
  border: 0px;
  background: #cbdae1;
  outline: none;

  color: black;
  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
    Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
  font-style: normal;
  font-weight: 500;
  font-size: 14px;
  line-height: 16px;
}

.input-phone[type="text"]:focus {
  border: 2px solid #ecd1c2;
}

.input-email[type="email"] {
  position: absolute;
  width: 245px;
  height: 51px;
  left: calc((100vw / 1.5) - 55px);
  top: 535px;

  box-sizing: border-box;
  border: 0px;
  background: #cbdae1;
  outline: none;

  color: black;
  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
    Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
  font-style: normal;
  font-weight: 500;
  font-size: 14px;
  line-height: 16px;
}

.input-email[type="email"]:focus {
  border: 2px solid #ecd1c2;
}

.input-email1[type="email"] {
  position: absolute;
  width: 245px;
  height: 51px;
  left: calc((100vw / 1.5) - 55px);
  top: 635px;

  box-sizing: border-box;
  border: 0px;
  background: #cbdae1;
  outline: none;

  color: black;
  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
    Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
  font-style: normal;
  font-weight: 500;
  font-size: 14px;
  line-height: 16px;
}

.input-email1[type="email"]:focus {
  border: 2px solid #ecd1c2;
}

.button {
  position: absolute;
  width: 130px;
  height: 38px;
  left: calc(100vw / 1.5);
  top: 730px;

  border: 1px solid #f0e6db;
  background-color: #ecd1c5;
  box-sizing: border-box;
  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);

  color: #68390d;
  text-align: center;
  font-size: 15px;
  font-weight: normal;
  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
    Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;

  opacity: 0.9;
  transition: 0.3s;
  cursor: pointer;
}

.button:hover {
  opacity: 1;
}

.button:active {
  background-color: #ecd1c2;
  box-shadow: 0 1px #666;
  transform: translateY(3px);
}

.side-panel {
  position: absolute;
  text-align: center;
  height: 100vh;
  width: 20vw;
  background: #cbdae1;
}

.logo {
  position: relative;
  width: 100px;
  height: 100px;
  top: 130px;
}

.nav {
  position: relative;
  top: 160px;

  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
    Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
  font-style: normal;
  font-weight: 400;
  font-size: 14px;
  line-height: 16px;

  color: #68390d;
}

ul {
  margin: 1;
  padding: 0;
}

li a {
  display: block;
  color: #68390d;
  padding: 8px 16px;
  text-decoration: none;
}

li a.active {
  background-color: #ecd1c2;
  color: #68390d;
}

li a:hover {
  background-color: #ecd1c2;
  color: #68390d;
}

.profile-pic {
  position: relative;
  width: 135px;
  height: 135px;
  top: 270px;
  left: calc(20vw / 2);
  border-radius: 75px;
}

.account-panel {
  position: absolute;
  text-align: center;
  justify-content: center;
  align-items: center;
  height: 100vh;
  width: 47vw;
  background: #62808e;
}

.name {
  position: absolute;
  width: 121px;
  height: 18px;
  top: 400px;
  left: calc((100vw / 3) - 55px);

  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
    Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
  font-style: normal;
  font-weight: 700;
  font-size: 24px;
  line-height: 28px;

  color: #e3d4c6;
}

.last-password {
  position: absolute;
  width: 230px;
  height: 21px;
  top: 450px;
  left: calc((100vw / 3) - 110px);

  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
    Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
  font-style: normal;
  font-weight: 400;
  font-size: 14px;
  line-height: 16px;

  color: #e3d4c6;
}
