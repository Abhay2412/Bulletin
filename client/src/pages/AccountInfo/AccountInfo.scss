@import "../../styles/colors.scss";

.account {
  height: 100%;

  &-panel-container {
    background: $color-lynch;
    height: 100%;
    width: 100%;
  }
  &-panel {
    height: 100%;
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;

    &-name {
      color: #e3d4c6;
      font-size: 2rem;
      font-weight: bold;
    }

    &-last-password {
      color: #e3d4c6;
      font-size: 1rem;
    }
  }

  &-edit {
    &-title {
      color: $color-brown;
    }
    &-grid-item {
      width: 60%;
      min-width: 200px;
    }

    &-grid-item > p {
      color: $color-brown;
    }

    &-text-field {
      margin-top: -10px;
      width: 100%;
      min-width: 200px;
    }
  }

  .save-button {
    background-color: $color-pink;
    color: $color-brown;
    width: 40%;
    min-width: 150px;
    margin-top: 50px;
  }
}
