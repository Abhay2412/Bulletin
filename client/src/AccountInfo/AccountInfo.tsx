import React from "react";
import profile from "./profile.svg";
import logo from "./logo.svg";
import "./AccountInfo.css";

const AccountInfo = () => {
    return (
        <div className="Account">
            <header className="Account-background">
                <div className="account-panel">
                    <img src={profile} className="profile-pic"></img>
                    <h1 className="name">John Doe</h1>
                    <h1 className="last-password">Last Password Change: 20/01/2022</h1>
                </div>

                <div className="side-panel">
                    <img src={logo} className="logo"></img>
                    <ul className="nav">
                        <li><a href="#">My Profile</a></li>
                        <li><a className="active" href="">Account Information</a></li>
                        <li><a href="">Personalization</a></li>
                        <li><a href="">Groups</a></li>
                        <li><a href="">Support Us</a></li>
                    </ul>
                </div>


                {/* <img src={profile} className="Profile-pic"></img>
                <h1 className="primary">John Doe</h1>
                <h1 className="primary1">Calgary, Alberta</h1>
                <h1 className="primary2">Last Login: 20/02/2022</h1>
                <h1 className="primary3">Brief Overview</h1>
                <h1 className="primary4">Name</h1>
                <h1 className="primary5">Birthday</h1>
                <h1 className="primary6">Email</h1>

                <form>
                    <input type="text" id="overview" placeholder="Tell everyone about yourself (max 150 characters)" className="input"></input>
                    <input type="text" id="name" placeholder="John Doe" className="input1"></input>
                    <input type="date" id="birthday" max="2012-12-31" className="input-date"></input>
                    <input type="email" id="email" placeholder="johndoe@gmail.com" className="input-email"></input>
                </form> */}
            </header>
        </div>
    );
};

export default AccountInfo;