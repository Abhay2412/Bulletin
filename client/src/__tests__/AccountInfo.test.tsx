import React from "react";
import AccountInfo from "../pages/AccountInfo/AccountInfo";
import { configure, mount, ReactWrapper } from "enzyme";
import renderer from "react-test-renderer";
import Adapter from "@wojtekmaj/enzyme-adapter-react-17";
import { store } from "../store";
import { Provider } from "react-redux";
import { BrowserRouter } from "react-router-dom";

configure({ adapter: new Adapter() });

describe("Testing <AccountInfo /> Component", () => {
  it("should render properly", () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <BrowserRouter>
            <AccountInfo />
          </BrowserRouter>
        </Provider>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("should change password field value", () => {
    const wrapper: ReactWrapper = mount(
      <Provider store={store}>
        <BrowserRouter>
          <AccountInfo />
        </BrowserRouter>
      </Provider>
    );
    wrapper
      .find("input")
      .find("#account-info-password-field")
      .simulate("change", {
        target: { value: "12345678" },
      });
    expect(
      wrapper.find("input").find("#account-info-password-field").get(0).props
        .value
    ).toEqual("12345678");
  });

  it("should change confirm password field value", () => {
    const wrapper: ReactWrapper = mount(
      <Provider store={store}>
        <BrowserRouter>
          <AccountInfo />
        </BrowserRouter>
      </Provider>
    );
    wrapper
      .find("input")
      .find("#account-info-confirm-password-field")
      .simulate("change", {
        target: { value: "12345678" },
      });
    expect(
      wrapper.find("input").find("#account-info-confirm-password-field").get(0)
        .props.value
    ).toEqual("12345678");
  });

  it("should change phone number field value", () => {
    const wrapper: ReactWrapper = mount(
      <Provider store={store}>
        <BrowserRouter>
          <AccountInfo />
        </BrowserRouter>
      </Provider>
    );
    wrapper
      .find("input")
      .find("#account-info-phone-field")
      .simulate("change", {
        target: { value: "+1 (921) 569-2927" },
      });
    expect(
      wrapper.find("input").find("#account-info-phone-field").get(0).props.value
    ).toEqual("+1 (921) 569-2927");
  });

  it("should change alternative email field value", () => {
    const wrapper: ReactWrapper = mount(
      <Provider store={store}>
        <BrowserRouter>
          <AccountInfo />
        </BrowserRouter>
      </Provider>
    );
    wrapper
      .find("input")
      .find("#account-info-alternative-email-field")
      .simulate("change", {
        target: { value: "test@domain.com" },
      });
    expect(
      wrapper.find("input").find("#account-info-alternative-email-field").get(0)
        .props.value
    ).toEqual("test@domain.com");
  });

  it("should have confirm password error", () => {
    const wrapper: ReactWrapper = mount(
      <Provider store={store}>
        <BrowserRouter>
          <AccountInfo />
        </BrowserRouter>
      </Provider>
    );

    wrapper
      .find('input[type="password"]')
      .find("#account-info-password-field")
      .simulate("change", {
        target: { value: "12345678" },
      });

    wrapper
      .find('input[type="password"]')
      .find("#account-info-confirm-password-field")
      .simulate("change", {
        target: { value: "" },
      });

    wrapper.find("button").find(".save-button").simulate("click");
    expect(
      wrapper
        .find("p")
        .find("#account-info-confirm-password-field-helper-text")
        .text()
    ).toEqual("Please re-type your password");
  });

  it("should show side drawer", () => {
    const wrapper: ReactWrapper = mount(
      <Provider store={store}>
        <BrowserRouter>
          <AccountInfo />
        </BrowserRouter>
      </Provider>
    );
    wrapper.find("button").find("#side-drawer-button").simulate("click");
    expect(wrapper.find("div").find("#side-drawer-id").exists()).toBeTruthy();
  });
});
