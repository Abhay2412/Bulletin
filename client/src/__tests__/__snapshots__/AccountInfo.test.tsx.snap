// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <AccountInfo /> Component should render properly 1`] = `
Array [
  <button
    className="MuiButtonBase-root MuiIconButton-root makeStyles-drawerButton-10"
    disabled={false}
    id="side-drawer-button"
    onBlur={[Function]}
    onClick={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex={0}
    type="button"
  >
    <span
      className="MuiIconButton-label"
    >
      <svg
        aria-hidden={true}
        className="MuiSvgIcon-root MuiSvgIcon-fontSizeLarge css-tzssek-MuiSvgIcon-root"
        data-testid="ArrowForwardIosIcon"
        focusable="false"
        style={
          Object {
            "color": "#68390D",
          }
        }
        viewBox="0 0 24 24"
      >
        <path
          d="M6.23 20.23 8 22l10-10L8 2 6.23 3.77 14.46 12z"
        />
      </svg>
    </span>
  </button>,
  <div
    className="account"
  >
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-item"
      style={
        Object {
          "height": "100%",
        }
      }
    >
      <div
        className="MuiGrid-root account-panel-container MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-5 MuiGrid-grid-lg-5 MuiGrid-grid-xl-5"
      >
        <div
          className="account-panel"
        >
          <img
            className="profile-pic"
            src="profile.svg"
          />
          <h1
            className="account-panel-name"
          >
            
          </h1>
          <h1
            className="account-panel-last-password"
          >
            Last Login:
             
          </h1>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-direction-xs-column MuiGrid-align-items-xs-center MuiGrid-justify-content-xs-center MuiGrid-grid-xs-12 MuiGrid-grid-md-7 MuiGrid-grid-lg-7 MuiGrid-grid-xl-7"
        style={
          Object {
            "height": "100%",
            "width": "100%",
          }
        }
      >
        <h1
          className="account-edit-title"
        >
          Account
        </h1>
        <div
          className="MuiGrid-root account-edit-grid-item MuiGrid-item"
        >
          <p>
            Change Your Password
          </p>
          <div
            className="MuiFormControl-root MuiTextField-root makeStyles-textField-1"
          >
            <label
              className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-filled"
              data-shrink={false}
              htmlFor="account-info-password-field"
              id="account-info-password-field-label"
            >
              Enter your new password
            </label>
            <div
              className="MuiInputBase-root MuiFilledInput-root MuiFilledInput-underline MuiInputBase-formControl"
              onClick={[Function]}
            >
              <input
                aria-invalid={false}
                autoFocus={false}
                className="MuiInputBase-input MuiFilledInput-input"
                disabled={false}
                id="account-info-password-field"
                onAnimationStart={[Function]}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                required={false}
                type="password"
                value=""
              />
            </div>
            
          </div>
        </div>
        <div
          className="MuiGrid-root account-edit-grid-item MuiGrid-item"
        >
          <p>
            Confirm New Password
          </p>
          <div
            className="MuiFormControl-root MuiTextField-root makeStyles-textField-1"
          >
            <label
              className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-filled"
              data-shrink={false}
              htmlFor="account-info-confirm-password-field"
              id="account-info-confirm-password-field-label"
            >
              Confrim your new password
            </label>
            <div
              className="MuiInputBase-root MuiFilledInput-root MuiFilledInput-underline MuiInputBase-formControl"
              onClick={[Function]}
            >
              <input
                aria-invalid={false}
                autoFocus={false}
                className="MuiInputBase-input MuiFilledInput-input"
                disabled={false}
                id="account-info-confirm-password-field"
                onAnimationStart={[Function]}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                required={false}
                type="password"
                value=""
              />
            </div>
            
          </div>
        </div>
        <div
          className="MuiGrid-root account-edit-grid-item MuiGrid-item"
        >
          <p>
            Add Phone Number
          </p>
          <div
            className="MuiFormControl-root MuiTextField-root makeStyles-textField-1"
          >
            <label
              className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-filled"
              data-shrink={false}
              htmlFor="account-info-phone-field"
              id="account-info-phone-field-label"
            >
              Provide your phone number
            </label>
            <div
              className="MuiInputBase-root MuiFilledInput-root MuiFilledInput-underline MuiInputBase-formControl"
              onClick={[Function]}
            >
              <input
                aria-invalid={false}
                autoFocus={false}
                className="MuiInputBase-input MuiFilledInput-input"
                disabled={false}
                id="account-info-phone-field"
                onAnimationStart={[Function]}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                required={false}
                type="text"
                value=""
              />
            </div>
            
          </div>
        </div>
        <div
          className="MuiGrid-root account-edit-grid-item MuiGrid-item"
        >
          <p>
            Primary Email Address
          </p>
          <div
            className="MuiFormControl-root MuiTextField-root makeStyles-textField-1"
          >
            <label
              className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-filled Mui-disabled Mui-disabled"
              data-shrink={false}
            >
              Primary Email
            </label>
            <div
              className="MuiInputBase-root MuiFilledInput-root MuiFilledInput-underline Mui-disabled Mui-disabled MuiInputBase-formControl"
              onClick={[Function]}
            >
              <input
                aria-invalid={false}
                autoFocus={false}
                className="MuiInputBase-input MuiFilledInput-input Mui-disabled Mui-disabled"
                disabled={true}
                onAnimationStart={[Function]}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                required={false}
                type="text"
                value=""
              />
            </div>
            
          </div>
        </div>
        <div
          className="MuiGrid-root account-edit-grid-item MuiGrid-item"
        >
          <p>
            Alternative Email Address
          </p>
          <div
            className="MuiFormControl-root MuiTextField-root makeStyles-textField-1"
          >
            <label
              className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-filled"
              data-shrink={false}
              htmlFor="account-info-alternative-email-field"
              id="account-info-alternative-email-field-label"
            >
              Alternative Email
            </label>
            <div
              className="MuiInputBase-root MuiFilledInput-root MuiFilledInput-underline MuiInputBase-formControl"
              onClick={[Function]}
            >
              <input
                aria-invalid={false}
                autoFocus={false}
                className="MuiInputBase-input MuiFilledInput-input"
                disabled={false}
                id="account-info-alternative-email-field"
                onAnimationStart={[Function]}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                required={false}
                type="text"
                value=""
              />
            </div>
            
          </div>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item"
        >
          <button
            className="MuiButtonBase-root MuiButton-root MuiButton-contained save-button"
            disabled={false}
            onBlur={[Function]}
            onClick={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex={0}
            type="button"
          >
            <span
              className="MuiButton-label"
            >
              Save
            </span>
          </button>
        </div>
      </div>
    </div>
  </div>,
]
`;
